BUILD

```diff
@@ -127,6 +127,125 @@ tfrt_cc_binary(
     ],
 )
 
+tfrt_cc_library(
+    name = "one_hot_encoder_cc",
+    srcs = [
+        "lib/linalg/sparse_vector.cc",
+        "lib/feature/one_hot_encoder.cc",
+    ],
+    hdrs = [
+        "include/clink/linalg/sparse_vector.h",
+        "include/clink/feature/one_hot_encoder.h",
+    ],
+    alwayslink_static_registration_src = "lib/kernels/static_registration.cc",
+    visibility = [":friends"],
+    deps = [
+        "@tf_runtime//:hostcontext",
+        "//protos:OneHotEncoderCC",
+    ],
+)
+
+tfrt_cc_binary(
+    name = "one_hot_encoder_jna",
+    srcs = [
+        "lib/feature/one_hot_encoder_jna.cc",
+    ],
+    linkshared = True,
+    visibility = [":friends"],
+    deps = [
+        ":one_hot_encoder_cc",
+        "@tf_runtime//:hostcontext_alwayslink"
+    ],
+)
+
+java_import(
+    name = "flink-ml",
+    jars = [
+        "java-lib/lib/flink-ml-core-0.1-SNAPSHOT.jar",
+        "java-lib/lib/flink-ml-lib_2.11-0.1-SNAPSHOT.jar",
+    ],
+)
+
+java_library(
+    name = "one_hot_encoder_java",
```

Comment:
Do we need to create one rule for every algorithm's dependencies? Can we follow Flink ML's approach?

Reply:
I'll merge all such rules into `clink_kernels` rules.